docs/sources/reference/commandline/cli.md

```diff
@@ -463,8 +471,8 @@ Currently supported options of `zfs`:
 
        $ docker -d -s zfs --storage-opt zfs.fsname=zroot/docker
```

Comment:
Looks like this line isn't rendered as code-block. Possibly needs more indenting

(Not introduced by your changes)